docs/blocks-storage/compactor.md

```diff
@@ -10,7 +10,7 @@ slug: compactor
 The **compactor** is an service which is responsible to:
 
 - Compact multiple blocks of a given tenant into a single optimized larger block. This helps to reduce storage costs (deduplication, index size reduction), and increase query speed (querying fewer blocks is faster).
-- Keep the per-tenant bucket index updated. The bucket index is used by [queriers](./querier.md) and [store-gateways](./store-gateway.md) to discover new blocks in the storage.
+- Keep the per-tenant bucket index updated. The [bucket index](./bucket-index.md) is used by [queriers](./querier.md) and [store-gateways](./store-gateway.md) to discover new blocks in the storage.
```

Comment:
In the bucket-index.md only queriers are mentioned. Let's make it consistent (either way is fine, but store-gateways support is not yet there).

Reply:
My plan is to update doc for store-gateway once support will be there. I will do it in the store-gateway PR.

Reply:
My point is that this document already mentions store-gateways, but bucket-index.md doesn't, which may be confusing. Either mention it at both places, or not at all?

Reply:
Oh right. I removed the mention to store-gateway. I will update it in the store-gateway PR.